### DISCUSSION TITLE
DSM connect issue

### DISCUSSION THREAD

**Discussion by mingil** [007revad/Synology_enable_M2_volume#186] [Q&A]
hello. I'm tried making nvme ssd as storage but now i can not connect to DSM. Only docker and webdav works. 

Here is the link: https://github.com/007revad/Synology_enable_M2_volume/issues/161
https://github.com/007revad/Synology_enable_M2_volume?tab=readme-ov-file#confirmed-working-on

Here are the commands I entered in SSH 

sudo -i

curl -kLO https://raw.githubusercontent.com/PeterSuh-Q3/tcrp-addons/main/nvmevolume-onthefly/syno_enable_m2_volume.sh

chmod +x syno_enable_m2_volume.sh

./syno_enable_m2_volume.sh

After that, i can connect to docker or webdav, but not DSM nor smb. 

I took someone else's advice and entered the command cp -vf /lib64/libhwcontrol.so.1.bak /lib64/libhwcontrol.so.1, but it's still the same. 

I was wondering if there is any way to get back to the way it was before. 

<img width="323" alt="20240529_101306" src="https://github.com/007revad/Synology_enable_M2_volume/assets/75480100/420ab323-3f5d-43bb-af00-884ef171f113">
<img width="328" alt="20240528_231859" src="https://github.com/007revad/Synology_enable_M2_volume/assets/75480100/b13ad363-6126-4901-8fbd-33ba11ba2abd">

**Comment by 007revad** [comment]
You need to reboot after editing usr/lib/libhwcontrol.so.1, or restoring usr/lib/libhwcontrol.so.1 from a backup.
